src/main/java/Duke.java

```diff
@@ -1,10 +1,77 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    public static final int MAX_SIZE = 100;
+    private static Task[] taskList = new Task[MAX_SIZE];
+    private static int taskCount = 0;
+
     public static void main(String[] args) {
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        showIntroduction();
+
+        do{
+            line = in.nextLine();
+            if(line.contains("list")) {
+                System.out.println("____________________________________________________________");
```

Comment:
Maybe this could be refactored with some other print statements to a function so that do not need to write this long statement everytime.